openadmet/models/eval/regression.py

```diff
@@ -109,10 +114,7 @@ def evaluate(
             self.use_wandb = use_wandb
 
         for task_id in range(n_tasks):
-            t_true = y_true[:, task_id]
-            t_pred = y_pred[:, task_id]
-            # remove Nan values
-            t_true, t_pred = mask_nans(t_true, t_pred)
```

Comment:
@dwwest I think the loss of `mask_nans` is responsible for the failing test here. Be careful! I think order matters here IIRC.